src/home/org.freedesktop.home1.policy

```diff
@@ -69,4 +69,13 @@
                 </defaults>
         </action>
 
+        <action id="org.freedesktop.home1.inhibit-suspend">
+                <description gettext-domain="systemd">Inhibit lock on suspend</description>
+                <message gettext-domain="systemd">Authentication is required to inhibit locking a user's home area on system suspend.</message>
```

Comment:
maybe change "locking" to "automatic lock"?

Reply:
better?